### PR TITLE
Improving source settings styles

### PR DESCRIPTION
**Pull request by elizabetdev**
source-settings

### CODE REVIEW

**Comment by cchaos** [comment]
The horizontal layout looks great. But I think the word "Metric" gets really repetitive.



**Two other things to think about.** 

1. We don't usually like to have panels within panels. Is there another way to divide or section off each Metric? Horizontal rule, light background color, etc?
2. Thoughts about using a toggle for the custom label? Where the toggle is in the right hand column, and only when the toggle has been swtiched on, the input appears, probably taking up the full width of the form row (not horizontal layout).

**Comment by elizabetdev** [comment]
The new design:

Screenshot 2019-10-01 at 19 08 01

File: x-pack/legacy/plugins/maps/public/components/_index.scss [hunk 1]
```diff
@@ -2,6 +2,26 @@
   margin-bottom: $euiSizeS;
 }
 
+.mapMetricEditorPanel__metricEditor {
```
Comment (elizabetdev) [inline x-pack/legacy/plugins/maps/public/components/_index.scss:5]: @cchaos I noticed that all the custom CSS is inside the `_index.scss`. Should I move them to their own files and then import on `_index.scss`?

Comment (cchaos) [reply x-pack/legacy/plugins/maps/public/components/_index.scss:5]: It **should** be moved to its own file. But in the interest of time, I'm going to approve this PR as is and you can follow up with a new branch that maybe turns on linting and fixes up the file structures?

File: x-pack/legacy/plugins/maps/public/components/_index.scss [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+  padding: $euiSizeM 0;
+  border-top: 1px solid $euiColorLightShade;
```
Comment (cchaos) [inline x-pack/legacy/plugins/maps/public/components/_index.scss]: Mostly FYI, we do have an overall variable for this which is 
```suggestion
  border-top:  $euiBorderThin;
```

File: x-pack/legacy/plugins/maps/public/components/metrics_editor.js [hunk 3]
```diff
@@ -25,31 +25,34 @@ export function MetricsEditor({ fields, metrics, onChange, allowMultipleMetrics,
...
+              iconType="trash"
+              size="xs"
+              color="danger"
+              onClick={onRemove}
+              aria-label={i18n.translate('xpack.maps.metricsEditor.deleteMetricAriaLabel', {
+                defaultMessage: 'Delete metric',
+              })}
+            >
+              <FormattedMessage
+                id="xpack.maps.metricsEditor.deleteMetricButtonLabe"
```
Comment (cchaos) [inline x-pack/legacy/plugins/maps/public/components/metrics_editor.js]:
```suggestion
                id="xpack.maps.metricsEditor.deleteMetricButtonLabel"
```

**Review by cchaos — APPROVED** [review]
Looks great, just had a couple suggestions. But once those are fixed, go ahead and merge this PR. Then you can bring your before/after screenshots into https://github.com/elastic/kibana/pull/47043 and pull it out of draft status.

File: x-pack/legacy/plugins/maps/public/components/_index.scss (hunk 1 above)
Comment (elizabetdev) [reply x-pack/legacy/plugins/maps/public/components/_index.scss:5]: Yes, no problem!

**Pull request merged**